src/dra/dra.h

```diff
@@ -539,12 +539,13 @@ extern Unkstruct_80102CD8 D_801379AC;
 extern s32 D_801379B0;
 extern s32 D_801379B8;
 extern s32 D_801379BC;
+extern SVECTOR D_801379C0;
 extern SVECTOR D_801379C8;
 extern VECTOR D_801379D0;
-extern VECTOR D_801379E0;
-extern VECTOR D_80137B20;
-extern SVECTOR D_80137CA0;
-extern SVECTOR D_80137D40;
+extern VECTOR D_801379E0[];
+extern VECTOR D_80137B20[];
```

Comment:
```suggestion
extern VECTOR D_80137B20[0x18];
```
from the loop. You will also need to adjust the stubs as you specified a length of 20 there instead of 24/0x18